### DISCUSSION TITLE
Dynamic Github Actions Dynamic Name

### DISCUSSION THREAD

**Discussion by schr4der** [community/community#48438] [Enterprise] [answered]
### Select Topic Area

Product Feedback

### Body

I'd like to make dynamic names with GitHub 'run-name' option in the workflow file. I've seen it work in regular GitHub actions, but I am trying to use it with an enterprise repo. Has anyone had any luck?

**Comment by jsoref** [comment, marked as answer]
it's available in https://docs.github.com/en/enterprise-server@3.8/actions/using-workflows/workflow-syntax-for-github-actions#run-name

You're presumably using 3.7 or older.
